src/component/variantPage/TherapeuticImplication.tsx

```diff
@@ -170,6 +170,7 @@ class TherapeuticImplication extends React.Component<
             >
                 <a href={oncokbUrl} target="_blank" rel="noopener noreferrer">
                     OncoKB&nbsp;<i className="fas fa-external-link-alt"></i>
+                    &nbsp;*
```

Comment:
Not a big deal but I prefer using spans over `&nbsp;` to add whitespace. It is a bit easier to read:
```suggestion
                    <span>OncoKB </span>
                    <i className="fas fa-external-link-alt"></i>
                    <span> *</span>
```